setup.py

```diff
@@ -42,10 +42,9 @@
     ],
     extras_require={
         'docs': [
-                'sphinx==1.2.3',  # autodoc was broken in 1.3.1
-                'sphinxcontrib-napoleon',
+                'sphinx',  # autodoc was broken in 1.3.1
```

Comment:
is the pin to 1.2.3 no longer necessary?

Reply:
the docs generate for me using the latest sphinx - this likely needs testing from another computer to make sure the docs generate everywhere